server/testdriver/core/src/main/java/org/infinispan/server/test/core/ContainerInfinispanServerDriver.java

```diff
@@ -155,41 +152,10 @@ protected void start(String name, File rootDir, String configurationFile) {
                .label("name", "Infinispan Server")
                .label("version", Version.getVersion())
                .label("release", Version.getVersion())
-               .label("architecture", "x86_64")
-               .user("root");
+               .label("architecture", "x86_64");
 
          if (!prebuiltImage) {
-            if (OS.getCurrentOs() != OS.WINDOWS) {
-               // We need to remap the UID/GID of the jboss user inside the container to the ones of the user outside so that files created on volume mounts have the correct ownership/permissions
-               String uid = runProcess("id", "-u");
-               String gid = runProcess("id", "-g");
-
-               builder
-                     .run("/bin/sh", "-c", "if [ -x \"$(command -v usermod)\" ]; then usermod -u " + uid +" jboss; fi")
-                     .run("/bin/sh", "-c", "if [ -x \"$(command -v groupmod)\" ]; then groupmod -g " + gid + " jboss; fi");
-            }
-            builder
-                  .copy("build", INFINISPAN_SERVER_HOME)
-                  .run("chown", "-R", "jboss:jboss", INFINISPAN_SERVER_HOME)
-                  .user("jboss");
-         }
-         // Copy the resources to a location from where they can be added to the image
-         try {
```

Comment:
@tristantarrant I didn't find a use case for this? Could you please help me on that ?

Reply:
This is needed when using this out-of-tree. It must be kept

Reply:
What means `using this out-of-tree`? What should be the test that must fail if we don't have this code?

Reply:
I have added the code

Reply:
Can you please try running your changes with https://github.com/tristantarrant/infinispan-playground-testdriver ?

Reply:
I was not able to create a cluster until https://github.com/tristantarrant/infinispan-playground-testdriver/pull/3

I have fixed `ContainerInfinispanServerDriver`
The main changes were:
* `public static final String IMAGE_USER = "185";`
* `builder.run("usermod -u " + IMAGE_USER + " jboss");`
The prebuilt image has the user 185, in this case, now we are also setting the user `185`

Reply:
I'd prefer doing this dynamically as there is no guarantee that the user id will stay constant and might also change depending on upstream/downstream/version/etc

Reply:
I have changed the user to `200` and also removed:
```
if (!prebuiltImage) {
  builder.run("usermod -u " + IMAGE_USER + " jboss");
}
```
It means that, if we built the image on the fly, will have a user `200`
If we use a prebuilt image, we are setting the user to `200`

When executing `chown` or `chmod` with an UID, Linux won't fail. ( This was a surprise for me)

I have executed the test in https://github.com/tristantarrant/infinispan-playground-testdriver and it passed.